website/docs/working-with-containers/registries/index.md

```diff
@@ -7,13 +7,38 @@ keywords: [podman desktop, podman, containers, registries]
 hide_table_of_contents: false
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Preconfigured from './\_preconfigured-registry.md'
+import Custom from './\_custom-registry.md'
+import Verification from './\_verification-private-registry.md'
+
 # Working with container registries
 
 Podman Desktop has access to public registries by default.
 
-You might want to configure Podman Desktop to:
+You might want to configure Podman Desktop to authenticate to:
+
+- A pre-configured registry:
+
+  - Red Hat Quay
+  - Docker Hub
+  - GitHub
+  - Google Container registry
```

Comment:
List is missing periods.

Reply:
When have that rule: list item require a period when it is a sentence, and no period when it is not a sentence. Unfortunately that is in the *IBM styleguide* that is not publicly available. Searching for a public reference for that rule.